src/meta/src/hummock/manager/tests.rs

```diff
@@ -643,111 +508,19 @@ async fn test_hummock_manager_basic() {
         HummockVersionId::MAX
     );
     for _ in 0..2 {
-        hummock_manager.unpin_version(context_id_1).await.unwrap();
-        assert_eq!(
-            hummock_manager.get_min_pinned_version_id().await,
-            HummockVersionId::MAX
-        );
-
         // should pin latest because u64::MAX
-        let version = match hummock_manager
-            .pin_version(context_id_1, HummockVersionId::MAX)
-            .await
-            .unwrap()
-        {
-            Payload::VersionDeltas(_) => {
-                unreachable!("should get full version")
-            }
-            Payload::PinnedVersion(version) => version,
-        };
+        let version = hummock_manager.get_current_version().await;
         assert_eq!(version.id, FIRST_VERSION_ID + 1);
-        assert_eq!(
-            hummock_manager.get_min_pinned_version_id().await,
-            FIRST_VERSION_ID + 1
-        );
     }
 
     commit_one(epoch, hummock_manager.clone()).await;
     // epoch += 1;
 
     for _ in 0..2 {
         // should pin latest because deltas cannot contain INVALID_EPOCH
-        let version = match hummock_manager
-            .pin_version(context_id_2, INVALID_EPOCH)
-            .await
-            .unwrap()
-        {
-            Payload::VersionDeltas(_) => {
-                unreachable!("should get full version")
-            }
-            Payload::PinnedVersion(version) => version,
-        };
+        let version = hummock_manager.get_current_version().await;
         assert_eq!(version.id, FIRST_VERSION_ID + 2);
-        // pinned by context_id_1
-        assert_eq!(
-            hummock_manager.get_min_pinned_version_id().await,
-            FIRST_VERSION_ID + 1
-        );
     }
-
-    // ssts_to_delete is always empty because no compaction is ever invoked.
```

Comment:
Could we pin_version explicitly here, to keep these tests related to vacuum.

Reply:
just resumed `test_hummock_manager_basic`